digitaltwins/service/src/service.json

```diff
@@ -2148,13 +2568,15 @@
             "TooManyModulesOnDevice",
             "ConfigurationCountLimitExceeded",
             "DigitalTwinModelCountLimitExceeded",
+            "InterfaceNameCompressionModelCountLimitExceeded",
```

Comment:
>InterfaceNameCompressionModelCountLimitExceeded [](start = 13, length = 47)

Pray what is a "Compression"Model?

Reply:
I pray with you...

---
In reply to: [449330230](https://github.com/Azure/azure-iot-sdk-node/pull/836#discussion_r449330230) [](ancestors = 449330230)